nutrition_labels/utils.py

```diff
@@ -0,0 +1,86 @@
+import re
+import pandas as pd
+from bs4 import BeautifulSoup
+from sklearn.metrics import confusion_matrix
+
+# Common synopsis/description fields which mean 'No data given'
+not_available_aliases = {
+        'Not available': None,
```

Comment:
Are you going to switch to our internal data labs one at some point?

Reply:
yep :) Will add an issue so it's not forgotten